frontend/src/pages/PokerPage.vue

```diff
@@ -1,17 +1,284 @@
 <template>
```

Comment:
Since this is the first time you used vue I am not reviewing this, instead I think we should sit together and do a pair programming session to make the UI for poker nicer.